routes/api/contacts.js

```diff
@@ -120,7 +145,48 @@ router.put("/:contactId", async (req, res, next) => {
         }
     } catch (err) {
         console.log(err.message);
+        res.status(500).json({
+            status: "failure",
+            code: 500,
+            message: err.message,
+        });
     }
 });
 
+router.patch("/:contactId/favorite", async (req, res, next) => {
+    try {
+        const body = {
+            favorite: req.query.favorite,
+        };
+        const result = await updateStatusContact(req.params.contactId, body);
+        if (result && result.status !== 400 && result !== 400) {
+            res.json({
+                status: "success",
+                code: 200,
+                data: result,
+            });
+            return;
+        }
+        if (!result) {
+            res.status(404).json({
+                status: "failure",
+                code: 404,
+                message: `Not found`,
+            });
+            return;
+        }
+        res.status(400).json({
+            // status: "failure",
+            // code: 400,
+            message: result.message || "missing field favorite",
+        });
+    } catch (err) {
+        console.log(err.message);
```

Comment:
```suggestion
        console.log(err);
```
Robienie console log błędu jest po to żeby tobie jako deweloperowi pokazać co się wydarzyło i pełniejsza informacja jest bardziej przydatna bo zawiera też dużo więcej informacji